pkg/db/db_local/backup.go

```diff
@@ -325,7 +327,7 @@ func runRestoreUsingList(ctx context.Context, info *RunningDBInstanceInfo, listF
 		// the database name
 		fmt.Sprintf("--dbname=%s", info.Database),
 		// connection parameters
-		"--host=localhost",
+		fmt.Sprintf("--host=%s", utils.GetFirstListenAddress(info.ListenAddresses)),
```

Comment:
This is always in a local installation. Prefer `localhost`

Reply:
As discussed in Slack, I'll update the PR to use `127.0.0.1` for all database maintenance tasks.

Reply:
Done.